src/Impact.php

```diff
@@ -804,6 +804,9 @@ public static function loadLibs(): void
     {
         echo Html::css('public/lib/cytoscape.css');
         echo Html::script("public/lib/cytoscape.js");
+        echo Html::script("public/lib/cytoscape-popper.js");
+        echo Html::script("public/lib/floating-ui-core.min.js");
+        echo Html::script("public/lib/floating-ui-dom.min.js");
```

Comment:
```suggestion
```
All of these should be in the `lib/bundles/cytoscape.js` so they get built together into `public/lib/cytoscape.js`.